python/egglog/egraph.py

```diff
@@ -577,23 +577,25 @@ def _generate_class_decls(  # noqa: C901,PLR0912
             decl = FunctionDecl(special_function_name, builtin=True, egg_name=egg_fn)
             decls.set_function_decl(ref, decl)
             continue
-
-        _, add_rewrite = _fn_decl(
-            decls,
-            egg_fn,
-            ref,
-            fn,
-            locals,
-            default,
-            cost,
-            merge,
-            on_merge,
-            mutates,
-            builtin,
-            ruleset=ruleset,
-            unextractable=unextractable,
-            subsume=subsume,
-        )
+        try:
+            _, add_rewrite = _fn_decl(
+                decls,
+                egg_fn,
+                ref,
+                fn,
+                locals,
+                default,
+                cost,
+                merge,
+                on_merge,
+                mutates,
+                builtin,
+                ruleset=ruleset,
+                unextractable=unextractable,
+                subsume=subsume,
+            )
+        except ValueError as e:
+            raise ValueError(f"Error processing {cls_name}.{method_name}: {e}") from e
```

Comment:
```suggestion
            raise ValueError(f"Error processing {cls_name}.{method_name}") from e
```

The `from e` should already give the previous stack trace.

Reply:
Or explicitly pull out the message.